plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/MapPart.java

```diff
@@ -68,10 +70,16 @@
      */
     public abstract IStatusLineManager getStatusLineManager();
 
-    // /**
-    // * EditDomain used to control the current tool.
-    // * @return
-    // */
-    // public abstract MapEditDomain getEditDomain();
+    /**
+     * Returns the DropHandler from this editor.
+     * 
+     * @return Returns the DropHandler from this editor.
+     */
+    public UDIGDropHandler getDropHandler();
+
+    // helper methods for tools
+    boolean isTesting();
```

Comment:
You may use the /** comment instead of // ?

Reply:
agree since it is "API" a valid javadoc would be great